assets/js/src/marketing_optin_block/block.tsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable react/react-in-jsx-scope */
+/* eslint-disable import/no-unresolved */
```

Comment:
We really don't like `eslint-disable` statements. You can check our [contributing documentation](https://github.com/mailpoet/mailpoet/blob/master/CONTRIBUTING.md#disabling-linting-rules). Is this really necessary? We can see the WooCommerce modules are defined in weback. We use a similar approach in our premium plugin but we don't need to disable Eslint import check.